### PR TITLE
Ghost bar!!!!

### PR DESCRIPTION
**Pull request by Aidenkrz**
## About the PR
Adds ghost bar :) Resolves https://github.com/Goob-Station/Goob-Station/issues/453

TODO:

- [X] Spawn ghost map bar on round start
- [X] Add button to ghost UI to spawn at ghost bar
- [x] Warning popup before spawning
- [x] Only allow players to spawn once to prevent them from creating piles of bodies (or delete their old body first?)
- [x] Cleanup map.

## Media

https://github.com/user-attachments/assets/f6f4b6b0-7387-4bbe-bc59-5e2347a6dce9

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: Ghost bar!

### CODE REVIEW

File: Content.Server/Goobstation/Ghostbar/GhostBarSystem.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    private void OnRoundStart(RoundStartingEvent ev)
+    {
+        var mapUid = _mapSystem.CreateMap(out var mapId, true);
+        var options = new MapLoadOptions
+        {
+            LoadMap = true,
+        };
+        _mapLoader.TryLoad(mapId, "Maps/Goobstation/Nonstations/ghostbar.yml", out _, options);
+        _mapSystem .SetPaused(mapId, false);
+    }
```
Comment (misandrie) [inline Content.Server/Goobstation/Ghostbar/GhostBarSystem.cs:43]:
```suggestion
    const string MapPath = "Maps/Goobstation/Nonstations/ghostbar.yml";
    private void OnRoundStart(RoundStartingEvent ev)
    {
        _mapSystem.CreateMap(out var mapId);
        var options = new MapLoadOptions { LoadMap = true };

        if (_mapLoader.TryLoad(mapId, MapPath, out _, options))
            _mapSystem.SetPaused(mapId, false);
    }
```

File: Content.Server/Goobstation/Ghostbar/GhostBarSystem.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+using Robust.Shared.Map;
+using Robust.Shared.Prototypes;
+using Robust.Server.Maps;
+using Robust.Shared.Random;
+using Content.Shared.Ghost;
+using Content.Server.Ghostbar.Components;
+using Content.Server.Mind;
+using Content.Shared.Roles.Jobs;
+
+namespace Content.Server.Ghostbar;
```
Comment (misandrie) [inline Content.Server/Goobstation/Ghostbar/GhostBarSystem.cs:16]:
```suggestion
namespace Content.Server.Goobstation.Ghostbar;
```

**Review by misandrie — CHANGES_REQUESTED** [review]
Fix indents troller

File: Content.Server/Goobstation/Ghostbar/Components/GhostBarSpawnComponent.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+namespace Content.Server.Ghostbar.Components;
```
Comment (misandrie) [inline Content.Server/Goobstation/Ghostbar/Components/GhostBarSpawnComponent.cs:1]:
```suggestion
namespace Content.Server.Goobstation.Ghostbar.Components;
```

**Review by misandrie — COMMENTED** [review]
also this

**Comment by misandrie** [comment]
also test fail is related

**Comment by Aidenkrz** [comment]
> also test fail is related

test fail is fixed when i cleanup the map

**Comment by Verbalase** [comment]
it’s real

**Pull request merged**